supabase/migrations/20241127203353_searchable_index_col.sql

```diff
@@ -0,0 +1,36 @@
+ALTER TABLE topics
+    ADD COLUMN searchable_index_col tsvector;
+
+CREATE INDEX search_idx ON topics USING gin(searchable_index_col);
+
+CREATE OR REPLACE FUNCTION private.update_searchable_index_col()
+    RETURNS TRIGGER
+    AS $$
```

Comment:
Consider installing the `pgFormatter` extension to format this code better 👍 https://marketplace.visualstudio.com/items?itemName=bradymholt.pgformatter

Reply:
I have it, this is how it looks after formatting it with `pgFormatter` :)